CHANGES.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 2.2.0 (unreleased)
 ------------------
 
+- #1990 Issue creating worksheet with method specific template
```

Comment:
Please be more specific. Something like "Fix items not filtered by Worksheet Template's method in Add analyses"

Reply:
done